src/components/ProjectSettingsTabs/NotificationsTab/getFormSchema.ts

```diff
@@ -0,0 +1,72 @@
+import { Field } from "@rjsf/core";
+import { SpruceFormProps } from "components/SpruceForm";
+import { CardFieldTemplate } from "components/SpruceForm/FieldTemplates";
+import widgets from "components/SpruceForm/Widgets";
+import { radioBoxOptions } from "../utils";
+import { DefaultSubscriptionsField } from "./Fields/DefaultSubscriptionsField";
+
+export const getFormSchema = (
+  useRepoSettings: boolean,
+  repoData?: any
```

Comment:
This can be typed to the tab's FormState in `NotificationsTab/types.ts`

Reply:
thanks, fixed